backend/dataall/modules/omics/api/mutations.py

```diff
@@ -0,0 +1,18 @@
+"""The module defines GraphQL mutations for Omics Pipelines"""
+
+from dataall.base.api import gql
+from .resolvers import *
+
+createOmicsRun = gql.MutationField(
+    name='createOmicsRun',
+    type=gql.Ref('OmicsRun'),
+    args=[gql.Argument(name='input', type=gql.NonNullableType(gql.Ref('NewOmicsRunInput')))],
+    resolver=create_omics_run,
+)
+
```

Comment:
If possible avoid string references `gql.Ref('NewOmicsRunInput')` and use the class instead `NewOmicsRunInput`